10up/brightcove-video-connect
# Add missing #mute field in toggleShortcodeGeneration function

### Description of the Change

This change adds #mute field in `toggleShortcodeGeneration` function. 

### Alternate Designs

### Benefits

When the user changes the value of field Mute in the Gutenberg editor, the button Update is enabled to save the change.

### Possible Drawbacks

### Verification Process

1. Add the Brightcove block
2. Save the page/post using the Update button of the editor
3. Edit the Brightcove block and change the Mute field
4. Check if the Update button gets enabled

### Checklist:






### Applicable Issues



### Changelog Entry

Fixed the Mute field in the block editing
